deployment/postgres-init-db/sql/views/hasura/hasura_event_logs.sql

```diff
@@ -0,0 +1,60 @@
+create function hasura.get_event_logs(_trigger_name text)
+returns table (
+  model_id int,
+  model_name text,
+  model_version text,
+  triggering_user text,
+  delivered boolean,
+  success boolean,
+  tries int,
+  created_at timestamp,
+  next_retry_at timestamp,
+  status int,
+  error jsonb,
+  error_message text,
+  error_type text
+)
+stable
+security invoker
+language plpgsql as $$
+begin
+  return query (
+    select
+      (el.payload->'data'->'new'->>'id')::int as model_id,
+      el.payload->'data'->'new'->>'name' as model_name,
+      el.payload->'data'->'new'->>'version' as model_version,
+      el.payload->'session_variables'->>'x-hasura-user-id' as triggering_user,
+      el.delivered,
+      eil.status is not distinct from 200 as success, -- is not distinct from to catch `null`
+      el.tries,
+      el.created_at,
+      el.next_retry_at,
+      eil.status,
+      (eil.response -> 'data'->> 'message')::jsonb as error,
```

Comment:
Non-blocking: Why have both the error and its fields in this view?

Reply:
The fields picked out are the ones that are most likely to contain the most relevant info for debugging the failure, but the overall `error` field is there in case there's additional information that wasn't in those fields that are important. I included `error` since these views are the only insight most users will be able to have into these event tables.